examples/deployments/terraform-gcp-github-actions/container.tf

```diff
@@ -0,0 +1,42 @@
+module "gce-container" {
+  source = "terraform-google-modules/container-vm/google"
+
+  container = {
+    image = "${var.region}-docker.pkg.dev/${var.project}/${var.registry}/${var.image}:latest" 
```

Comment:
@Jaglyser, quick question about the images. Do you think this is important to add in the README - e.g. how to create your project registry, login to it and push images?

Reply:
Yeah, I can do that! 
Didn't think of it but could of course be useful to someone who hasn't done so before :)

Reply:
Updated the readme!
Have a look at the 6th step :D

Reply:
Looks great. I am sure it will save valuable time for people simply by explaining that they should put their images in the GCP registry for their specific project.

Reply:
Yeah, I hope so! 
Anything else you think should be added?